google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -192,7 +213,21 @@ public void onStart(StreamController controller) {
     @Override
     public void onResponse(StreamingPullResponse response) {
       channelReconnectBackoffMillis.set(INITIAL_CHANNEL_RECONNECT_BACKOFF.toMillis());
+      boolean isExactlyOnceDeliveryEnabled =
+          response.getSubscriptionProperties().getExactlyOnceDeliveryEnabled();
+      if (enableExactlyOnceDelivery.get() != isExactlyOnceDeliveryEnabled) {
+        enableExactlyOnceDelivery.set(isExactlyOnceDeliveryEnabled);
+        messageDispatcher.setEnableExactlyOnceDelivery(isExactlyOnceDeliveryEnabled);
+
+        // Update modack extension defaults if applicable
+        if (defaultStreamAckDeadline) {
+          messageDispatcher.setMessageDeadlineSeconds(
```

Comment:
Does this get sent over to the server in the next request?

Reply:
Discussed offline yesterday. It does get sent over